test_scenarios/kubernetes/3idx1sh1cm-pvc/splunk-indexer-statefulset-persistent.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: indexer
+  #namespace: splunk
```

Comment:
Commenting out the `namespace` here, breaks the `dnsConfig` below, or at least I believe it will. I haven't actually tested it. >.<

Reply:
i removed the commented namespace. README walkthrough expects the command to include the ```-n <namespace>``` flag